Model/Config/Alipay.php

```diff
@@ -1,5 +1,5 @@
 <?php
-namespace Omise\Payment\Model\Config\Offsite;
+namespace Omise\Payment\Model\Config;
```

Comment:
@jacekstan I got the point that we want to make a generic class that share its functionality between `offsite` and `offline` payment methods as now we have `Gateway/Request/PaymentAPMBuilder.php`.
But not sure why we need to move the classes out from `Offsite` folder here?

Reply:
That is very simple structure with small classes that have 1 file only, I don't see the reason why it cannot be in one folder at first place.

Now I will add tesco, and will have to create `offline` folder, I just don't see the point to complicate that structure for those few configs.